src/pages/index.tsx

```diff
@@ -81,7 +48,7 @@ const Home: NextPage = () => {
           <Headline label="お知らせ" headlineTypes={headlineFontSize} />
         </div>
         <div className="news-card-top-page">
-          <NewCardList newsDataList={dummyCardList} />
+          <NewCardList newsDataList={newsCardData} />
```

Comment:
[NITS] タイポしてますねw
```suggestion
          <NewsCardList newsDataList={newsCardData} />
```

割とこういうのを避けるために、named exportするようにするのが割と最近のトレンドかもです。

```tsx
// NewsCard.tsx
export const NewsCard = () => {}

// pages/index.tsx
import { NewsCard } from 'NewsCard'
```

こうしておくことで、もしコンポーネント名を`NewsCard`から`InformationCard`とかに変更したときにimportしているファイルでコンパイルエラーになって、コンポーネント名は変えているはずなのに使っている場所では修正してなかったみたいなことをなくすことができます。

```tsx
// ファイル名とコンポーネント名を変えた時
// InformationCard.tsx
export const InformationCard = () => {}

// pages/index.tsx
// NewsCardはexportされていないのでエラーになる
import { NewsCard } from 'InformationCard'
```